rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -139,7 +139,7 @@ def __init__(self): # pylint: disable=too-many-statements
             directory = "win-x64"
             libname = "rtiddsconnector"
             post = "dll"
-            additional_lib = "msvcr120"
```

Comment:
not sure why we didn't provide the `.dll` suffix previously. Looks to me like it is necessary at least.

Reply:
I don't think you need the extension. The OS calls usually work with and without.